2022-RapidReact/src/main/java/frc/robot/RobotContainer.java

```diff
@@ -4,24 +4,36 @@
 
 package frc.robot;
 
+import frc.robot.subsystems.Climber;
+import frc.robot.Constants;
 import edu.wpi.first.wpilibj.GenericHID;
 import edu.wpi.first.wpilibj.XboxController;
+import frc.robot.commands.ClimberExtend;
+import frc.robot.commands.ClimberRetract;
 import frc.robot.commands.ExampleCommand;
 import frc.robot.subsystems.ExampleSubsystem;
 import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.button.JoystickButton;
 
 /**
  * This class is where the bulk of the robot should be declared. Since Command-based is a
  * "declarative" paradigm, very little robot logic should actually be handled in the {@link Robot}
  * periodic methods (other than the scheduler calls). Instead, the structure of the robot (including
  * subsystems, commands, and button mappings) should be declared here.
  */
+
+private final Climber climber = new Climber();
+
 public class RobotContainer {
   // The robot's subsystems and commands are defined here...
   private final ExampleSubsystem m_exampleSubsystem = new ExampleSubsystem();
 
   private final ExampleCommand m_autoCommand = new ExampleCommand(m_exampleSubsystem);
 
+  private final JoystickButton buttonRb = new JoystickButton(RobotContainer.controller, Constants.BUTTON_RB_ID);
+  private final JoystickButton buttonLb = new JoystickButton(RobotContainer.controller, Constants.BUTTON_LB_ID);
```

Comment:
O André tinha me dito que ele acredita haver uma classe que possa mapear os botões do controle; mas ainda to pesquisando sobre ela, então não sei exatamente como funciona.